services/winddirection.md

```diff
@@ -0,0 +1,20 @@
+# Wind direction
+
+A sensor that measures wind direction.
+
+    identifier: 0x186be92b
+    extends: _sensor
+
+## Registers
+
+    ro wind_direction: u16 ° @ reading
```

Comment:
maybe overkill but u16.16 for future-proofing?

Reply:
does not hurt... but the spark fun sensor precision was around 40deg